pysph/tools/cull.py

```diff
@@ -0,0 +1,92 @@
+"""Culls files in a given directory, one in every 'c' files is spared.
+
+The specified directory can contain other directories that house the output
+ files; files in all these directories will be culled, sparing one in every
+ 'c' files. Note that DELETION IS PERMANENT.
+"""
+
+from pysph.tools.binder import find_sim_dirs, find_dir_size
+from pysph.solver.utils import get_files
+import os
+import sys
+import argparse
+import numpy as np
+
+
+def cull(src_path, c):
+
+    src_path = os.path.abspath(src_path)
+    sim_paths_list = find_sim_dirs(src_path)
+
+    initial_size = find_dir_size(src_path)
+
+    for path in sim_paths_list:
+        files = get_files(path)
+        del_files = [
+            files[i] for i in np.setdiff1d(
```

Comment:
You don't need numpy here for this, the standard library `set` will do the trick just as well.
```
In [1]: set(range(5)) - set(range(0, 5, 2))
Out[1]: {1, 3}
```

Reply:
Changed it in the latest commit.